docs/v1/P4Runtime-Spec.adoc

```diff
@@ -6036,6 +6036,11 @@ message only includes the `p4runtime_api_version` string field. This field must
 be set to the full semantic version string cite:[SemVer] corresponding to the
```

Comment:
The previous sentence sentence needs an update. Currently it says: "... the `CapabilitiesResponse`
message only includes the `p4runtime_api_version` string field"